code/game/objects/items.dm

```diff
@@ -1824,6 +1824,16 @@
 
 	return src
 
+///Called when an item is added to a slot of a fishing rod.
+/obj/item/proc/on_fishing_rod_slotted(obj/item/fishing_rod/rod, slot)
+	SHOULD_CALL_PARENT(TRUE)
+	SEND_SIGNAL(src, COMSIG_ITEM_FISHING_ROD_SLOTTED, rod, slot)
+
+///Called when an item is removed from a slot of a fishing rod
+/obj/item/proc/on_fishing_rod_unslotted(obj/item/fishing_rod/rod, slot)
+	SHOULD_CALL_PARENT(TRUE)
+	SEND_SIGNAL(src, COMSIG_ITEM_FISHING_ROD_UNSLOTTED, rod, slot)
```

Comment:
This has no place on /obj/item, put these on smth like /obj/item/fishing and repath (and make an UpdatePaths script) for lured and lines and what not uses it

Reply:
I can move these and some of the other procs in a different file, perhaps in the fishing module, but I don't agree with this, kind of. The bait slot can take any kind of item as long as they have the bait trait, even if it seldom has any additional effect.